src/components/grapedown/render.js

```diff
@@ -69,8 +70,12 @@ class Renderer {
   }
 
   render(props) {
+    let text = props.text
```

Comment:
I think, render method can be simplified.
```
{
  this.props = props
  if (navigator.platform === 'Win32') {
    return this.renderer(jsEmoji.replace_unified(props.text))
  }
  return this.renderer(props.text)
}
```
Is this looks better?

Reply:
works for me, just made the change